lib/publishing/npm/publish-npm.sh

```diff
@@ -22,6 +22,11 @@ fi
 # NPM #
 #######
 
+DISTTAG=${DISTTAG:-""}
+if [ -n "${DISTTAG}" ]; then
+    DISTTAG="--tag=${DISTTAG}"
```

Comment:
I find it confusing that the semantics of the `DISTTAG` variable changes at this point.

Reply:
It's a matter of style and I rather not have to run this test again (it takes like 30 minutes).